modules/govuk_mysql/templates/usr/local/bin/xtrabackup_s3_restore.erb

```diff
@@ -52,27 +60,26 @@ else
   # the directory is a mount point so we need to copy
   echo "Backing up current dir"
   TEMPDIR=mysql-$(date '+%s')
```

Comment:
You should rename this to something other than `$TEMPDIR` as this is often already a value in the shell.